packages/@aws-cdk/aws-ecs/test/fargate/integ.lb-awsvpc-nw.ts

```diff
@@ -27,6 +27,7 @@ container.addPortMappings({
 const service = new ecs.FargateService(stack, "Service", {
   cluster,
   taskDefinition,
+  longArnEnabled: true
```

Comment:
It's good that you've tested this, but we will not be able to reproduce this on our non-long-ARN accounts :(

Reply:
So revert the integ test change I think. We'll make do with the one-off manual test and the unit test.

Reply:
Okay, I was wondering about this -- it makes me a little uncomfortable that we don't have a way to integ test this since it's dependent on AWS account settings state. I'm wondering if we a have a place to track things like this, especially for other contributors who want to develop CDK features -- another similar issue that came up is running the tests in different regions (fargate is not in all the same regions as ECS, e.g.).